backend/src/apiserver/server/run_server.go

```diff
@@ -499,7 +499,17 @@ func (s *RunServer) terminateRun(ctx context.Context, runId string, experimentId
 	return s.resourceManager.TerminateRun(ctx, runId)
 }
 
-// Terminates a run.
+// Restarts a run. Not exported.
```

Comment:
Maybe use "retry" to be more aligned with the method name?

"Not exported." is not needed and obvious since the function starts with the lowercase letter.

Reply:
Thank you so much! I fixed this and other comments in this file.